src/BaselineOfClap/BaselineOfClap.class.st

```diff
@@ -8,14 +8,21 @@ Class {
 BaselineOfClap >> baseline: spec [
 	<baseline>
 	spec for: #common do: [ spec
+		baseline: 'Okay' with: [ spec repository: 'github://cdlm/okay-st' ];
```

Comment:
This is not being integrated no?
I don't see any "Okay" code. Is that ok?

Then we can discuss if we want another framework or not for testing. We have already discussed in the past about introducing mocking frameworks in the base image.

Reply:
It's not integrated, I refactored the baseline so that the default and development groups don't include either Okay or the booklet examples.

Okay is intended to be an assertion / extreme validation framework. But maybe it's better to port Andreas's Assessments… and Exceptions need work to make them properly nestable

Reply:
> It's not integrated, I refactored the baseline so that the default and development groups don't include either Okay or the booklet examples.

ok